@codex please fix

```
 building client (vite) 
 ✓ tests/generateItemDependencies.test.ts (3 tests) 75ms
 ✓ tests/navigateWithRetry.test.ts (2 tests) 11ms
 ✓ tests/k8sEnvironmentOverlays.test.ts (2 tests) 4ms
 ✓ tests/noConflictMarkers.test.ts (1 test) 1205ms
   ✓ repository sanity ‑ no merge-conflict markers > contains no <<<<<<< or >>>>>>> markers  1203ms
 ✓ tests/svelteResolution.test.ts (4 tests) 427ms
 ✓ scripts/tests/questDependencies.test.ts (2 tests) 5ms
 ✓ tests/architectureNotesAccessibilityBaseline.test.ts (2 tests) 5ms
 ✓ tests/questDependencyReferences.test.ts (1 test) 38ms
 ✓ tests/questIdPathConsistency.test.ts (1 test) 25ms
 ✓ tests/svelteSubpathResolver.test.ts (2 tests) 6ms
 ✓ tests/serviceWorkerConfigCache.test.ts (1 test) 5ms
 ✓ tests/noDuplicateItemIds.test.ts (1 test) 5ms
 ✓ frontend/tests/docsMarkdownSafety.test.ts (1 test) 11ms
 ✓ tests/questReliabilityCoverage.test.ts (1 test) 129ms
 ✓ tests/v3ReleaseStateDoc.test.ts (3 tests) 7ms
 ✓ frontend/tests/questListClassifier.test.ts (3 tests) 5ms
 ✓ tests/changelogQuestBranchingNote.test.ts (1 test) 6ms
 ✓ tests/metricsEndpoint.test.ts (3 tests) 19ms
 ✓ tests/docsSearchFeatures.test.ts (2 tests) 8ms
 ✓ tests/questProcessRecoveryPaths.test.ts (1 test) 135ms
 ✓ tests/questsPolishPromptSections.test.ts (3 tests) 4ms
 ✓ tests/builtinQuestSchema.test.ts (1 test) 83ms
 ✓ tests/githubTokenPersistence.test.ts (1 test) 13ms
 ✓ frontend/src/components/__tests__/QuestLinkChips.spec.ts (2 tests) 166ms
 ✓ tests/spellcheckConfig.test.ts (1 test) 8ms
01:18:39 [build] Rearranging server assets...
01:18:39 [build] Server built in 33.48s
01:18:39 [build] Complete!
 ✓ tests/buildOutputValidation.test.ts (1 test) 40130ms
   ✓ build output validation > should not contain /src/scripts/ paths in built files  40126ms
 ✓ tests/pnpmFrontendFilter.test.ts (1 test) 14ms
 ✓ tests/scanSecretsScript.test.ts (2 tests) 84ms
 ✓ tests/helmChartReleaseVersion.test.ts (1 test) 5ms
 ✓ tests/changelogDocsAICoPilots.test.ts (1 test) 5ms
 ✓ tests/compactItemListHelpers.test.ts (2 tests) 6ms
 ✓ tests/userJourneysCoverage.test.ts (1 test) 4ms
 ✓ tests/customContentLoadTest.test.ts (2 tests) 1307ms
   ✓ runCustomContentLoadTest > uses default count when omitted  1280ms
      Tests  1 failed | 1042 passed (1043)
   Start at  01:16:57
   Duration  155.38s (transform 9.74s, setup 10.55s, collect 44.14s, tests 110.90s, environment 184.60s, prepare 34.55s)


Error: Error: npm run format:check failed.

> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
[warn] scripts/utils/ensure-playwright-browsers.js
[warn] tests/ensure-playwright-browsers.test.ts
[warn] Code style issues found in 2 files. Run Prettier with --write to fix.

 ❯ scripts/tests/prettierFormatting.test.ts:17:13


 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/frontend/scripts/utils/ensure-playwright-browsers.js
+++ b/frontend/scripts/utils/ensure-playwright-browsers.js
@@ -191,7 +191,8 @@ export async function ensurePlaywrightBrowsers(options = {}) {
 
     if (env.PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD === '1') {
         if (browser) {
-            const { hasChromium, hasHeadlessShell, executablePath } = getChromiumAssetStatus(browser);
+            const { hasChromium, hasHeadlessShell, executablePath } =
+                getChromiumAssetStatus(browser);
             if (!hasChromium) {
                 console.warn(
                     'PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 but Playwright chromium browser is missing. E2E tests may fail.'
@@ -237,7 +238,9 @@ export async function ensurePlaywrightBrowsers(options = {}) {
 
     const postInstallAssets = getChromiumAssetStatus(browser);
     if (!postInstallAssets.hasChromium || !postInstallAssets.hasHeadlessShell) {
-        const missingAsset = !postInstallAssets.hasChromium ? 'chromium executable' : 'chromium headless shell';
+        const missingAsset = !postInstallAssets.hasChromium
+            ? 'chromium executable'
+            : 'chromium headless shell';
         console.warn(
             `Playwright ${missingAsset} is still missing after installation. Tests may fail if browsers are unavailable.`
         );

--- a/frontend/tests/ensure-playwright-browsers.test.ts
+++ b/frontend/tests/ensure-playwright-browsers.test.ts
@@ -324,7 +324,6 @@ describe('ensurePlaywrightBrowsers', () => {
         expect(writeFileSyncMock).toHaveBeenCalled();
     });
 
-
     it('uses provided env for skip-download behavior', async () => {
         const chromiumPath = '/root/.cache/ms-playwright/chromium-1234/chrome';
         existsSyncMock.mockImplementation((target: string) => target === chromiumPath);

